godotengine/godot
# Add emit_changed() to Environment setters

Similar to #99729, added missing `emit_changed()` to the `Environment` resource setters.

Attached a MRP for testing:
[cam_env.zip](https://github.com/user-attachments/files/17965511/cam_env.zip)

Steps to reproduce / test:
1. Run the scene
2. Select the `Camera3D` node from the Scene Tree
3. Expand the `Environment` resource attached to the `Camera3D`
4. Modify `Environment` properties
5. Observe the `Output` console emitting a print statement whenever a value is changed.
